nautobot/circuits/models.py

```diff
@@ -215,22 +215,35 @@ class CircuitTermination(PrimaryModel, PathEndpoint, CableTermination):
     xconnect_id = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True, verbose_name="Cross-connect ID")
     pp_info = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True, verbose_name="Patch panel/port(s)")
     description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    cloud_network = models.ForeignKey(
+        to="cloud.CloudNetwork",
+        on_delete=models.PROTECT,
+        related_name="circuit_terminations",
+        blank=True,
+        null=True,
+    )
```

Comment:
Maybe move this up so it's adjacent to the `location` and `provider_network` field declarations?